File: calcu.py
#Operaciones basicas

valor1 = 5
valor2 = 5

suma = valor1 + valor2
resta = valor1 - valor2
multi = valor1 * valor2
division = valor1 / valor2
print("suma = ", suma)
print("resta = ", resta)
print("producto = ", multi)
print("division = ", division)

def fib(n):    
  a, b = 0, 1
  while a < n:
    print(a, end=' ')
    a, b = b, a+b
    print()
fib(1000)

File: calcu.php
<?php 
print_r("<ul>");
function suma(){
  $a = 8;
  $b = 6;
  $suma = $a + $b;
  print_r("<li>8+6=".$suma."</li>");
    }
suma();

function resta(){
  $a = 8;
  $b = 6;
  $resta = $a - $b;
  print_r("<li>8-6= ".$resta."</li>");
}
resta();

function multi(){
  $a = 8;
  $b = 6;
  $multi = $a * $b;
  print_r("<li>8x6=".$multi."</li>");
}
multi();

function div(){
  $a = 8;
  $b = 6;
  $div = $a/$b; 
  print_r("<li>8/6=".$div."</li>");
}
div();
print_r("</ul>");
?>
